.github/scripts/workflow_rerun/errors_to_look_for.json

```diff
@@ -70,5 +70,9 @@
     {
         "error_text": "Unable to make request: ECONNRESET",
         "ticket": 148702
+    },
+    {
+        "error_text": "json.decoder.JSONDecodeError: Unterminated string starting at",
```

Comment:
I fear the error message might be too generic and could be attributed to other bugs and not only to network-related problems. However, as the rerunner has an upper limit of reruns per pipeline, it might be fine.